client/components/AdoptionForm.js

```diff
@@ -0,0 +1,153 @@
+import React, { useState } from "react"
+import { Redirect } from "react-router-dom"
+import _ from "lodash"
+
+import ErrorList from "./ErrorList"
+
+const AdoptionForm = props => {
+  const [formData, setFormData] = useState({
+    name: "",
+    phoneNumber: "",
+    email: "",
+    homeStatus: "",
+    applicationStatus: "pending",
+    adoptablePetId: props.adoptablePetId
+  })
+  const [errors, setErrors] = useState({})
+  const [shouldRedirect, setShouldRedirect] = useState(false)
+
+  const handleChange = event => {
+    setFormData({
+      ...formData,
+      [event.currentTarget.name]: event.currentTarget.value
+    })
+  }
+
+  const addNewAdoptionApplication = async () => {
+    const { type, id } = props
+    try {
+      const response = await fetch(`/api/v1/adoptable-pets/${type}/${id}/`, {
```

Comment:
Submitting an adoption form should be going to `adoption_applications` for POSTing or creating a new adoption application, we're not creating a new adoptable pet